package.json

```diff
@@ -41,7 +41,8 @@
     "crypto-js": "3.1.2-2",
     "node-xmpp-client": "^3.0.0",
     "request": "^2.48.0",
-    "strophe.js": "^1.2.8",
+    "strophe.js": "1.2.12",
+    "webrtc-adapter": "^2.1.0",
```

Comment:
@dimaspirit  I mean will the following work:
```
<script src="quickblox.min.js"></script>
<script src="adapter.js"></script>
```
?

or there will be some conflicts?